examples/webgpu/script.js

```diff
@@ -0,0 +1,153 @@
+// Requests a WebGPU adapter and create a device from it.
+const adapter = await navigator.gpu.requestAdapter();
+const device = await adapter.requestDevice();
+
+// Get a WebGPU context from the canvas and configure it for use with the device.
+const canvas = document.getElementById('canvas');
+const context = canvas.getContext('webgpu');
+context.configure({
+  device,
+  format: navigator.gpu.getPreferredCanvasFormat(),
+  alphaMode: 'premultiplied',
+});
+
+// Set up the vertex and fragment shaders.
+const module = device.createShaderModule({
+  code: `
+    // Define the four corners of the square being rendered
+    var<private> pos : array<vec2f, 4> = array<vec2f, 4>(
```

Comment:
```suggestion
    var<private> pos = array(
```

Reply:
I didn't know about this bit of syntactic sugar! Thanks!

Reply:
Done